packages/start/server/babel.js

```diff
@@ -143,11 +144,11 @@ function transformServer({ types: t, template }) {
                   let statement = path.findParent(p => program.get("body").includes(p));
                   let serverIndex = state.servers++;
                   let hasher = state.opts.minify ? hashFn : str => str;
-                  const hash = hasher(
-                    state.filename.replace(state.opts.root, "").slice(1) + "/" + serverIndex
-                  );
+                  const fName = state.filename.replace(state.opts.root, "").slice(1)
 
-                  const route = `${INLINE_SERVER_ROUTE_PREFIX}/${hash}`;
```

Comment:
Wasn't this for RPC routing? I don't think we need to use `path` here.

Reply:
hash will contain os-specific delimiters, so path will be like `a\b\c/d/e`. It is useful to use this methods to have correct path joining